examples/create_geopandas_from_pandas.py

```diff
@@ -28,21 +27,15 @@
      'Longitude': [-58.66, -47.91, -70.66, -74.08, -66.86]})
 
 ###############################################################################
-# A ``GeoDataFrame`` needs a ``shapely`` object, so we create a new column
-# **Coordinates** as a tuple of **Longitude** and **Latitude** :
+# A ``GeoDataFrame`` needs a ``shapely`` object. We use geopandas
+# ``points_from_xy()`` to transform **Longitude** and **Latitude** into a list
+# of ``shapely.Point`` objects and set it as a ``geometry`` while creating the
+# ``GeoDataFrame``.
```

Comment:
Is it useful to still say that this is more or less equivalent to `[Point(x, y) for x, y in zip(df.Longitude, df.Latitude)]`, to give users some understanding what is happening behind the scenes?